openff/interchange/_tests/unit_tests/interop/lammps/export/test_lammps.py

```diff
@@ -62,3 +68,104 @@ def test_to_lammps_single_mols(self, mol, sage_unconstrained, n_mols):
                 "Torsion": 0.02 * unit.kilojoule_per_mole,
             },
         )
+
+    def test_unique_lammps_mol_ids(
+        self, smiles: str, sage_unconstrained: ForceField, sidelen: int = 0
+    ) -> bool:
+        """Test to see if interop.lammps.export._write_atoms() writes unique ids for each distinct Molecule"""
+        temp_lammps_path = (
+            Path.cwd() / "temp.lmp"
+        )  # NOTE: lammps writer doesn't currently support IO-like object passing, so need to intercept output with temporary file
```

Comment:
```suggestion
        )
```
Tracked in https://github.com/openforcefield/openff-interchange/issues/1001